### PR TITLE
[ticket/13952] Inject event dispatcher dependency to the tidy search cron task

### PR DESCRIPTION
**Pull request by rxu**
<a href="https://tracker.phpbb.com/browse/PHPBB3-13952">PHPBB3-13952</a>.

### CODE REVIEW

File: phpBB/phpbb/cron/task/core/tidy_search.php [hunk 1]
```diff
@@ -26,6 +26,7 @@ class tidy_search extends \phpbb\cron\task\base
 	protected $config;
 	protected $db;
 	protected $user;
+	protected $phpbb_dispatcher;
```
Comment (marc1706) [inline phpBB/phpbb/cron/task/core/tidy_search.php]: Could you maybe add a doc block for this? Even though the old ones don't have that, I like it if we at least add a doc block for newly added lines of code.

**Comment by rxu** [comment]
@marc1706 Updated.

**Comment by bantu** [comment]
@Nicofuma Hmm, I thought the conclusion was that this wasn't a problem.

**Comment by Nicofuma** [comment]
It isn't an issue if we call the constructor with to many arguments. not the contrary.

**Comment by bantu** [comment]
@Nicofuma Looks like I remember it wrong, or confused the two.

**Pull request merged**